gigascience/gigadb-website
# Relabel button that saves attribute in adminFile to avoid ambiguity

# Pull request for issue: #1858

This is a pull request for the following functionalities:

* Relabel save button to "save attribute" for more specificity
* Convert input into a button of type submit for better a11y support

![image](https://github.com/gigascience/gigadb-website/assets/143437854/3b10e25d-3d3b-4c92-8ce8-09b9612610d5)


## How to test?

- navigate to e.g. http://gigadb.gigasciencejournal.com/adminFile/update/id/446
- create new attr
- click on edit button on the new attr and check that save button label is updated

## Any issues with implementation?

To allow the parent "save" button to save changes in the edited attribute, the controller action needs to be updated

## Any changes to automated tests?

--
